src/hawkmoth/docstring.py

```diff
@@ -236,6 +236,22 @@ class TypeDocstring(Docstring):
     _indent = 1
     _fmt = '.. {domain}:type:: {name}'
 
+class TypeAliasDocstring(Docstring):
```

Comment:
> [...] especially since it does not support using in general but only its use for type aliases.

Thought the same, though I think no other use of `using` would produce a document-able cursor. Either way, I don't like it that much either and you make a good point. 'Type alias' it is ;)

Reply:
Also not sure. There is `using namespace`, `using enum`, and `using ParentClass::symbol` but I don't see why one would want these in the documentation.